webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableActions.js

```diff
@@ -17,7 +14,7 @@ import {
 
 export const fetchInsights = (queryParams = {}) => (dispatch, getState) => {
   const state = getState();
-  const { page, perPage, query, sortBy, sortOrder } = {
+  const { page, perPage, query, sortBy, sortOrder, isSelectAll = false } = {
```

Comment:
No need to set it here as the selector will always return a boolean value
```suggestion
  const { page, perPage, query, sortBy, sortOrder, isSelectAll } = {
```